lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/DriverStartHandler.java

```diff
@@ -71,7 +71,7 @@
   private DriverStartHandler(final Set<EventHandler<StartTime>> startHandler,
                              final Optional<Set<EventHandler<StartTime>>> restartHandlers,
```

Comment:
Orthogonal to the change at hand, but: Tang will always inject `Set<T>` parameters. If nothing is bound to them, they will be empty. Hence, you can write code assuming that the `Set<T>` exists. Loops over it will just not do anything.